src/main/java/com/spartronics4915/frc2019/subsystems/PanelHandler.java

```diff
@@ -28,19 +28,19 @@ public static PanelHandler getInstance()
 
     public enum WantedState
     {
-        RETRACT, EJECT
+        ARM_DOWN, ARM_UP, EJECT_ROCKET
     }
 
     private enum SystemState
     {
-        RETRACTING, EJECTING
+        ARM_DOWNING, ARM_UPING, EJECTING_ROCKET
     }
 
-    private WantedState mWantedState = WantedState.RETRACT;
-    private SystemState mSystemState = SystemState.RETRACTING;
+    private WantedState mWantedState = WantedState.ARM_UP;
+    private SystemState mSystemState = SystemState.ARM_UPING;
 
     private Solenoid mSolenoid = null;
-
+    private Solenoid mSolenoidArm = null;
```

Comment:
Change `SolenoidArm` to `ArmSolenoid`... Change `Solenoid` to `ExtendSolenoid` or `SlideSolenoid`.